src/js/dp/validation/validation.js

```diff
@@ -105,7 +105,7 @@ function getErrorSummary(errors, isPageError) {
 
   return `<div aria-labelledby="alert" role="alert" tabindex="-1" id="form-validation-panel" class="ons-panel ons-panel--error ons-u-mb-m ons-u-mt-m">
```

Comment:
You need to update the aria label too
```suggestion
  return `<div aria-labelledby="error-summary-title" role="alert" tabindex="-1" id="form-validation-panel" class="ons-panel ons-panel--error ons-u-mb-m ons-u-mt-m">
```